tee-worker/omni-executor/pumpx/src/pumpx_api/types.rs

```diff
@@ -143,3 +145,14 @@ pub struct NewLimitOrder {
 pub struct LimitOrder {
 	pub order_id: String,
 }
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Hmm should it be just `google_code`? (snake_case)

Reply:
Oh, yes. Thanks

Reply:
I also noticed that - they seem to mixing up camel and snake cases